introduction/supported-environments.adoc

```diff
@@ -24,7 +24,19 @@ This section highlights supported version updates in {PRODUCT} {PRODUCT_VERSION}
 // |Apache Kafka| RH AMQ Equivalent
 |===
 
-.Supported libraries in {PRODUCT} {PRODUCT_VERSION}
+.Supported libraries in {PRODUCT} {VERSION_901}
+[%header,cols=2]
+[%autowidth]
+|===
+|Library |Supported Version
+|{QUARKUS} | {QUARKUS_VERSION_SHORT_12_FINAL}
+|Kogito runtimes| 1.40.2.Final
+|Kogito add-ons| 1.40.2.Final
+|Drools | 8.40.1.Final
```

Comment:
it's not all libraries from those repos that we support. I think we have to articulate ore clear that we support a subset of libraries - and for those the versions are the ones listed here.